parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -39,23 +39,29 @@ installDeps() {
     # blobfuse2 is installed for all ubuntu versions, it is included in pkg_list
     # for 22.04, fuse3 is installed. for all others, fuse is installed
     # for 16.04, installed blobfuse1.3.7, for all others except 22.04, installed blobfuse1.4.5
-    pkg_list+=(blobfuse2=${BLOBFUSE2_VERSION})
+    pkg_list+=("blobfuse2=${BLOBFUSE2_VERSION}")
     if [ "${OSVERSION}" = "22.04" ] || [ "${OSVERSION}" = "24.04" ]; then
         pkg_list+=(fuse3)
     else
-        pkg_list+=(blobfuse=${BLOBFUSE_VERSION} fuse)
+        pkg_list+=("blobfuse=${BLOBFUSE_VERSION}" fuse)
     fi
 
     if [ "${OSVERSION}" = "24.04" ]; then
         pkg_list+=(irqbalance)
     fi
 
+    pkg_list+=("aznfs=0.3.15")
```

Comment:
0.3.15 is the correct version for all Ubuntu versions we support? (1804,2004,2204,2404)

Reply:
also, I'm assuming it's intended that we're not installing on mariner/azlinux SKUs?

Reply:
@Devinwong I'm assuming we aren't yet able to have renovate manage updates to packages like this, though would like to confirm

Reply:
@cvvz we only need this package in `2204,2404`